### DISCUSSION TITLE
Welcome to WPGraphQL Discussions!

### DISCUSSION THREAD

**Discussion by jasonbahl** [wp-graphql/wp-graphql#1447] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hashimwarren** [comment]
I look forward to participating here!

**Comment by hashimwarren** [comment]
Also I'm looking forward to 1.0

https://github.com/wp-graphql/wp-graphql/issues/1357

**Comment by AmineLahmary** [comment]
this is the qeury : query Posts {
 posts {
  nodes {
   id
  }
 }
} 
and this is the out put : 
{
 "errors": {
  "message": "Unexpected token '<', \"<html><bod\"... is not valid JSON",
  "stack": "SyntaxError: Unexpected token '<', \"<html><bod\"... is not valid JSON"
 }
}
i want to get data without been logged in
